src/psyclone/psyir/tools/definition_use_chains.py

```diff
@@ -759,6 +764,11 @@ def _compute_backward_uses(self, basic_block_list):
                         # Note that this assumes two symbols are not
                         # aliases of each other.
```

Comment:
Not strictly this PR but:
```
$ pytest --cov psyclone.psyir.tools.definition_use_chains --cov-report term-missing psyir/tools/definition_use_chains_*test.py
---------- coverage: platform linux, python 3.13.0-final-0 -----------
Name                                                                            Stmts   Miss  Cover   Missing
------------------------------------------------------------------------------------------------------------------
/home/me/Projects/PSyclone/src/psyclone/psyir/tools/definition_use_chains.py     417      3    99%   658-660
------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                  417      3    99%
```

Reply:
Not sure how that missing coverage slipped through, I've added something for it now.

Reply:
Ah codecov.io seemed to think that has coverage on head somehow, how strange. Its covered now but I guess something else in the code hit that.

Reply:
Yes, it will be something else in the test suite. The older the code, the poorer we are at achieving this but it does make coverage checking much easier when you're developing.